static_src/components/saving.jsx

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
What's the reason we couldn't combine this with the loading component, and allow that component to have text like "Saving"?